cpp/src/io/json/json_column.cu

```diff
@@ -809,6 +838,24 @@ void make_device_json_column(device_span<SymbolT const> input,
                     stream.value());
   }
 
+  // ignore all children of columns forced as string
+  for (auto const this_col_id : unique_col_ids) {
+    auto parent_col_id = column_parent_ids[this_col_id];
+    if (parent_col_id != parent_node_sentinel and forced_as_string_column[parent_col_id] == 1) {
+      forced_as_string_column[this_col_id] = 1;
+      ignore_vals[this_col_id]             = 1;
+    }
+    // Convert only mixed type columns as string (so to copy), but not its children
+    if (parent_col_id != parent_node_sentinel and forced_as_string_column[parent_col_id] == 0 and
+        forced_as_string_column[this_col_id] == 1)
+      column_categories[this_col_id] = NC_STR;
+  }
+  cudaMemcpyAsync(d_column_tree.node_categories.begin(),
```

Comment:
please use `cuda_memcpy_async` to allow the SM copy strategy.

Reply:
Replaced!